source/Willow-Bootstrap/BootstrapNavigationBar.class.st

```diff
@@ -58,30 +58,24 @@ BootstrapNavigationBar >> initializeStyledAs: aContainerClass brandedWith: aBran
 { #category : #rendering }
 BootstrapNavigationBar >> renderContentOn: aCanvas [
 
-	| nav |
-
-	nav := aCanvas navigation.
-	nav class: BootstrapCssStyles navbar.
-	componentCommand applyTo: nav on: aCanvas.
-	nav
+	aCanvas navigation
 		with: [ aCanvas div
-				class: containerClass;
 				with: [ self renderNavigationBarHeaderOn: aCanvas.
-					aCanvas render: navbarContent ] ]
+					aCanvas render: navbarContent ]
+				applying: containerCommand ]
+		applying: [ :nav | nav addClass bootstrap navbar + componentCommand ]
 ]
 
 { #category : #private }
 BootstrapNavigationBar >> renderNavigationBarHeaderOn: aCanvas [
 
 	aCanvas div
-		class: BootstrapCssStyles navbarHeader;
-		with: [ 
-			navbarContent identifyIn: aCanvas.
+		with: [ navbarContent identifyIn: aCanvas.
```

Comment:
Since we a have a long nested block, it might improve readability to use `apply: ; with: ... `